pyproject.toml

```diff
@@ -1,13 +1,80 @@
 [build-system]
-# Further build requirements - like numpy & cython - come from setup.py via
-# the PEP 517 interface.
 requires = [
-    "setuptools",
-    "wheel",
+    "Cython ~=0.29",
     "oldest-supported-numpy",
     "pkgconfig",
-    "Cython >=0.29; python_version<'3.8'",
-    "Cython >=0.29.14; python_version=='3.8'",
-    "Cython >=0.29.15; python_version>='3.9'",
+    "setuptools >=61",
 ]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "h5py"
+description = "Read and write HDF5 files from Python"
+authors = [
+    {name = "Andrew Collette", email = "andrew.collette@gmail.com"},
+]
+maintainers = [
+    {name = "Thomas Kluyver", email = "thomas@kluyver.me.uk"},
+]
```

Comment:
It certainly shouldn't be just me here. We should either get rid of the maintainers list altogether or add the other maintainers - at a minimum, @tacaswell, @aragilar and @ajelenak, though there are several others who've been given maintainer access at some point but aren't so active at present.

Reply:
I propose we keep this list to "people willing to deal with emails about h5py".

Reply:
I'm happy enough for my email to be there - we shouldn't just point everyone to Andrew. I imagine that dealing with emails will mostly mean telling people to use the issue tracker.

Reply:
Other maintainers (@aragilar @ajelenak @kif @aparamon ), please do say if you're happy for your name (and optionally email) to be in this list of maintainers. If I don't hear in the next few days, I'll merge this, though you're always welcome to add yourself later.

I'd rather be inclusive than exclusive in any kind of list of credits, so anyone who's made more than trivial contributions is welcome to be named as a maintainer if they'd like to be. But we won't share your real name or email on PyPI without your confirmation.